docs/guides/admin/docs/workflowoperationhandlers/prepareav-woh.md

```diff
@@ -11,7 +11,7 @@ If it there is one track with a certain flavor, the "encode" method is called wh
 using the same container and codec (-vcodec copy, -a codec copy), while the container format is determined by ffmpeg via
 the file's extension. The reason for doing this is that many media files are in a poor state with regard to their
 compatibility (most often, the stream's codec contains differing information from the container), so we are basically
-asking ffmepg to rewrite the whole thing, which will in many cases eliminate problems that would otherwhise occur later
+asking ffmepg to rewrite the whole thing, which will in many cases eliminate problems that would otherwise occur later
```

Comment:
Nitpicking:

```suggestion
asking FFmepg to rewrite the whole thing, which will in many cases eliminate problems that would otherwise occur later
```

FFmpeg is the projects name, ffmpeg is the executable. Thus, I think we ask FFmpeg but execute `ffmpeg`.

…not that anyone really cares :D

Reply:
I care!